src/Task1_S2.java

```diff
@@ -0,0 +1,48 @@
+public class Task1_S2 {
+
+    //    String-2 > doubleChar
+    //    Given a string, return a string where for every char in the original, there are two chars.
+    //    doubleChar("The") → "TThhee"
+    //    doubleChar("AAbb") → "AAAAbbbb"
+    //    doubleChar("Hi-There") → "HHii--TThheerree"
+
+    public static void main(String[] args) {
+        System.out.println(doubleChar("Ukraine"));
+    }
+
+    public static String doubleChar(String str) {
+
+        //Below is my code that runs well on IDEA, but it does not run on https://codingbat.com
+        //showing error "cannot find symbol repeat(int)".
+
+        if (str.length() == 0) {
+            return "";
+
+        } else {
+            String strDoubleFull = str.substring(0, 1).repeat(2);
+
+            for (int i = 1; i < str.length(); i++) {
+                String strDoubleChar = str.substring(i, i + 1).repeat(2);
+                strDoubleFull = strDoubleFull.concat(strDoubleChar);
+            }
+            return strDoubleFull;
+        }
+    }
+}
+
+//Below is the code that also runs well on https://codingbat.com.
```

Comment:
If this code passes all the tests, then it should be used, and the snippet via `repeat()` shouldn't be primary here